sdk/storage/azure-storage-file-datalake/tests/testcase.py

```diff
@@ -107,7 +107,7 @@ def enable_logging(self):
         handler = logging.StreamHandler()
         handler.setFormatter(logging.Formatter(LOGGING_FORMAT))
         self.logger.handlers = [handler]
-        self.logger.setLevel(logging.INFO)
+        self.logger.setLevel(logging.DEBUG)
```

Comment:
any particular reason?

Reply:
Our Logging policy works only when the logging level is set to DEBUG, so if it's set to INFO, even if we config `setting_real.py` and set `ENABLE_LOGGING = True` nothing will be logged

Reply:
Interesting :)